index.bs

```diff
@@ -1545,6 +1566,14 @@ that determine how the [=WebTransport session=] is established and used.
    preferred application-protocol and communicating it to the client is optional for the server.
    Servers might reject the request if a suitable protocol was not provided.
 
+: <dfn for="WebTransportOptions" dict-member>datagramsReadableMode</dfn>
+:: Optionally specifies an application's preference for using a default [=readable stream=]
+   or [=readable byte stream=] for incoming datagrams.
```

Comment:
```suggestion
:: Optionally specifies an application's preference for using a
   [=readable byte stream=] for incoming datagrams. Otherwise,
   a default [=readable stream=] is used. 
```